build.bat

```diff
@@ -10,4 +10,4 @@ IF EXIST paket.lock (
 	.paket\paket.exe install
 	IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
 )
-"packages\build\FAKE\tools\Fake.exe" "build\\scripts\\Targets.fsx" "cmdline=%*"
+"packages\build\FAKE.x64\tools\FAKE.exe" "build\\scripts\\Targets.fsx" "cmdline=%*"
```

Comment:
Curious why this was needed, did we run into any limitation?

Reply:
Yep: https://github.com/elastic/windows-installers/pull/102/commits/8c9b48108e71da69f6d1af949a76897fbdfbf936